homework46/src/components/Albums.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { fetchUrl, getUsersData } from "../App";
+import { Routes, Route } from "react-router-dom";
+import Photo from "./Photo";
+
+export default function Albums({ userId, isPhotos }) {
+    const [albums, setAlbums] = useState([])
+    const [photos, setPhotos] = useState([])
+
+    useEffect(() => {
+        getUsersData(`/albums`, setAlbums);
+    }, [])
+
+    useEffect(() => {
+        getUsersData('/photos', setPhotos)
```

Comment:
Read previous comment and don't pass setter as an argument, use promise instead of